cfpb/teachers-digital-platform
# Implemented separation of content and print ui

Short description explaining the high-level reason for the pull request

## Additions

- Created a series of components that take content in JavaScript JSON format and renders it on the FinalPrintPage.js 

## Testing

1. npm run build
1. Make sure to fill out the first 2 criterion on the Content criterion
   1. All three have been started in the content_data folder
   1. The idea is to create a JSON file in the content_data folder
      1. I recommend creating the content for each Dimension in a separate file
1. Use the existing FinalPrintPage.js as a guide
1. NOTE: the Content has a switcher component that swaps out for the correct one

## Notes
- I had to use dangerouslySetInnerHTML for styling in some of the content

## TODO
- I still need to come up with an option for the Quality text fields (actually the dangerslySetInnerHTML might work since that will show on the print page as plain text

## Review

- @dcmouyard @rrstoll 

[Preview this PR without the whitespace changes](?w=0)
